packages/b/boost/xmake.lua

```diff
@@ -70,6 +70,7 @@ package("boost")
 
     add_configs("all",          { description = "Enable all library modules support.",  default = false, type = "boolean"})
     add_configs("multi",        { description = "Enable multi-thread support.",  default = true, type = "boolean"})
+    add_configs("lto",          { description = "Enable lto for compiling.", default = false, type = "boolean"})
```

Comment:
lto is builtin config, you need not add it.

Reply:
I force-pushed for deleting the line `add_configs("lto"...`